api/src/utils/generics/lifecycle-hook-manager.ts

```
@@ -39,6 +39,20 @@ interface Registry {
 export class LifecycleHookManager {
   private static registry: Registry = {};
 
+  private static models = new Map<string, ModelDefinition>();
+
+  public static getModel(name: string) {
+    return this.models.get(name);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)